drivers/modem/hl78xx/Kconfig.hl78xx

```
@@ -57,33 +57,21 @@ config MODEM_HL78XX_12_FW_R6
 
 endif # MODEM_HL78XX_12
 
-config MODEM_HL78XX_UART_BUFFER_SIZES
-	int "The UART receive and transmit buffer sizes in bytes."
-	default 512
-
-config MODEM_HL78XX_CHAT_BUFFER_SIZES
-	int "The size of the buffers used for the chat scripts in bytes."
-	default 512
-
-config MODEM_HL78XX_USER_PIPE_BUFFER_SIZES
-	int "The size of the buffers used for each user pipe in bytes."
-	default 512
-
-config MODEM_HL78XX_RECV_BUF_CNT
-	int "The number of allocated network buffers"
-	default 30
-
-config MODEM_HL78XX_RECV_BUF_SIZE
-	int "The size of the network buffers in bytes"
-	default 128
-
-config MODEM_HL78XX_RX_WORKQ_STACK_SIZE
-	int "Stack size for the Sierra Wireless HL78XX driver modem driver work queue"
-	default 2048
-	help
-	  This stack is used by the work queue to pass off net_pkt data
-	  to the rest of the network stack, letting the rx thread continue
-	  processing data.
+config MODEM_HL78XX_AT_SHELL
+	bool "AT command shell"
+	help
+	  Enable AT command shell for the modem.
+if MODEM_HL78XX_AT_SHELL
+config MODEM_AT_SHELL_RESPONSE_TIMEOUT_MS
+	int "Timeout waiting for response to AT command in milliseconds"
+	default 500
+config MODEM_AT_SHELL_COMMAND_MAX_SIZE
+	int "Maximum size of AT command"
+	default 32
+config MODEM_AT_SHELL_RESPONSE_MAX_SIZE
+	int "Maximum size of AT response"
+	default 64
+endif # MODEM_HL78XX_AT_SHELL
```

Fix some formatting here. Should be blank lines between separate configs and the start of the if.


```suggestion
config MODEM_HL78XX_AT_SHELL
	bool "AT command shell"
	help
	  Enable AT command shell for the modem.

if MODEM_HL78XX_AT_SHELL

config MODEM_AT_SHELL_RESPONSE_TIMEOUT_MS
	int "Timeout waiting for response to AT command in milliseconds"
	default 500

config MODEM_AT_SHELL_COMMAND_MAX_SIZE
	int "Maximum size of AT command"
	default 32

config MODEM_AT_SHELL_RESPONSE_MAX_SIZE
	int "Maximum size of AT response"
	default 64

endif # MODEM_HL78XX_AT_SHELL
```